src/javascript/app/pages/trade/barriers.js

```diff
@@ -160,14 +161,75 @@ const Barriers = (() => {
         Defaults.remove('barrier', 'barrier_high', 'barrier_low');
     };
 
-    const validateBarrier = () => {
+    const showError = (barrier) => {
+        if (!barrier.classList.contains('error-field')) {
+            barrier.classList.add('error-field');
+        }
+        const error_node = barrier.parentNode.lastElementChild.firstElementChild;
+        if (error_node.classList.contains('invisible')) {
+            error_node.classList.remove('invisible');
+        }
+    };
+
+    const resolveError = (barrier) => {
+        if (barrier.classList.contains('error-field')) {
+            barrier.classList.remove('error-field');
+        }
+        const error_node = barrier.parentNode.lastElementChild.firstElementChild;
+        if (!error_node.classList.contains('invisible')) {
+            error_node.classList.add('invisible');
+        }
+    };
```

Comment:
it seems these two functions are exactly the same, except one uses remove, add and the other uses add, remove. could be worth combining them with a flag to hide/show the error.

Reply:
maybe it can have another param for `remove=True`?

Reply:
this is the one that I meant to comment let's leave as it is, let's leave the show error and hide error as two different functions